src/application/tasks/ventilation.c

```diff
@@ -23,11 +24,11 @@
  */
 static int start_current_breath_cycle = 0;
 
-/*
- * the time the current breath cycle was supposed to start
- */
-static int breath_cycle_duration = 125; //to do replace this value by the real one
-
+uint32_t calculate_breath_cycle_duration(){
+  uint32_t breath_cycle_duration = 0; // Variable initialization
+  uint32_t respiratory_rate = 15; // Retrieve clinician respiratory rate, THIS IS JUST A PLACEHOLDER FOR NOW
+  return breath_cycle_duration = 60000/respiratory_rate; // Calculate cycle duration

```

Comment:
make the computation in a separate line.

Reply:
for readability use spaces  between symbols 

```suggestion
  breath_cycle_duration = 60000 / respiratory_rate; // Calculate cycle duration

```

Reply:
the comment is not necessary